pkg/controller/installation/core_controller.go

```diff
@@ -1081,6 +1073,16 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 		return reconcile.Result{}, err
 	}
 
+	if instance.Spec.Variant == operator.TigeraSecureEnterprise {
+		managerInternalTLSSecret, err := certificateManager.GetCertificate(r.client, render.ManagerInternalTLSSecretName, common.OperatorNamespace())
+		if err != nil {
+			r.status.SetDegraded(operator.ResourceReadError, fmt.Sprintf("Error fetching TLS secret %s in namespace %s", render.ManagerInternalTLSSecretName, common.OperatorNamespace()), err, reqLogger)
+			return reconcile.Result{}, nil
+		} else if managerInternalTLSSecret != nil {
+			typhaNodeTLS.TrustedBundle.AddCertificates(managerInternalTLSSecret)
```

Comment:
What does typha need the manager tls secret for? Does it use voltron?

Reply:
We render only one bundle per namespace.
The way the code evolved over time before the bundle was introduced is that this typhaNodeTLS struct is passed along for configuring node, typha and other components. This is why the trustedBundle is also in this struct. It's a bit misleading.

Reply:
The cert internal-manager-tls needs to be in the bundle to have the kube controllers trust voltron for MCM requests, but Typha has no need for it.

Reply:
OK, so it's needed for kube controllers. I think I posed the question before about this, but does kube controllers need this? What requests is kube controllers (not es kube controllers) making to voltron?

Reply:
es-kube-controllers is the pod requiring access, not kube-controllers. Because it resides in the calico-system namespace, it is using the trusted bundle created by the core_controller.